### DISCUSSION TITLE
Change "Salary" to "Income"

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#166] [General Survey Design]
Self-employed folks don't necessarily have a salary (and even employees often have additional income streams), so "Income" is more inclusive.

**Comment by ShaineRosewel** [comment]
Instead of categories, can we also instead ask respondents for an average value? We could apply certain restrictions to the field to make sure that respondents enter a number. I think this will be helpful later in the presentation of results. What do you think, @SachaG?

**Reply by LeaVerou** [reply]
I think the idea is that a bracket may be more privacy-preserving than entering a specific number, and faster to enter (single click vs 5-6 keystrokes).

**Reply by SachaG** [reply]
Yes, the reason I'm in favor of brackets are:

1. privacy issue
2. some people might not know their exact income
3. faster to input
4. if we have a limited number of possible values it can also simplify the calculations

**Reply by LeaVerou** [reply]
Yeah, 1 and 3 is basically what I wrote. Not sure how common 2 is, don't they file taxes? I'm not sure I understand 4 either, since you can always get brackets from a number, so you can do the exact same calculations, it's just that you can do *more* if you want.

**Reply by ShaineRosewel** [reply]
I do agree with all your reasons. With that, what we have as responses later on in the results, are ordered categories, not values. By definition, we cannot take averages (which is one of what @LeaVerou mentioned as _more_ that can be done if we have values instead of categories).

**Reply by LeaVerou** [reply]
Hi folks, since this is separate from the discussion about the wording change from "salary" to "income" (and affects other questions too) I opened a new topic: https://github.com/Devographics/surveys/discussions/177

(and also turned it into a [blog post](https://lea.verou.me/blog/2023/numbers-vs-brackets/))

Looking forward to your thoughts there!
